src/libmongoc/tests/test-mongoc-client.c

```diff
@@ -3969,8 +3980,16 @@ test_client_install (TestSuite *suite)
    TestSuite_AddLive (suite,
                       "/Client/get_description/pooled",
                       test_mongoc_client_get_description_pooled);
-   TestSuite_AddLive (
-      suite, "/Client/descriptions", test_mongoc_client_descriptions);
+   TestSuite_AddLive (suite,
+                      "/Client/descriptions/single",
+                      test_mongoc_client_descriptions_single);
```

Comment:
This does not fail on single, ironically due to a bug in libmongoc: https://jira.mongodb.org/browse/CDRIVER-3716 which causes libmongoc not to send a $clusterTime in the first ismaster command to the arbiter.